roles/jboss_brms/tasks/main.yml

```diff
@@ -85,46 +100,76 @@
 
 - name: cat MANIFEST.MF files
   raw: cat '{{ item }}/META-INF/MANIFEST.MF' 2>/dev/null
-  register: jboss_brms_manifest_mf
+  register: internal_jboss_brms_manifest_mf
   ignore_errors: yes
   become: yes
   with_items: "{{ (business_central_candidates + decision_central_candidates + kie_server_candidates + internal_jboss_brms_kie_search_candidates.get('stdout_lines', []))|difference(['']) }}"
   when: 'user_has_sudo and jboss_brms'
 
+- name: set jboss_brms_manifest_mf fact
+  set_fact:
+    jboss_brms_manifest_mf: '{{ internal_jboss_brms_manifest_mf }}'
```

Comment:
should we use " instead of ' to be consistent? (for all of the set_facts i feel like we should pick one and be consistent)

Reply:
🤕

Reply:
😬

Reply:
I believe in your Cody.  You can make it consistent.